canonical-web-and-design/maas.io
# Create docs/api page manually

## Done

- Created a docs page at /docs/api which was generated from [this md file](https://github.com/CanonicalLtd/maas-docs/blob/master/en/api.md)

## QA

- Go to /docs/api and check that it reflects the content at https://github.com/CanonicalLtd/maas-docs/blob/master/en/api.md

## Issue / Card

Fixes canonical-web-and-design/MAAS-squad#1320
